tests/inventory

```diff
@@ -1,26 +1,199 @@
+#### ALL HOSTS AND CONTAINERS START HERE
+
 [all]
-localhost physical_host=localhost
+localhost ansible_become=True ansible_connection=local ansible_user=root
+infra1
```

Comment:
Gotcha. The reason I asked is because you have ``tests/host_vars/infra2.yml`` and ``tests/host_vars/infra3.yml`` in this PR